src/index.html

```diff
@@ -1,12 +1,172 @@
 <!DOCTYPE html>
 <html lang="en">
+
   <head>
     <meta charset="UTF-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
+
   <body>
-    <h1>Calendar</h1>
+
+    <div class="container">
+      <div class="cell">
+        <div class="cell__day">Mon</div>
+        <div class="cell__date">1</div>
```

Comment:
Create a markup for the calendar block with 31 days inside (only blocks)
DON'T add numbers in HTML (you will do it using CSS) (it needs to do it with @for)